react/src/components/SslCertificateError.js

```diff
@@ -0,0 +1,26 @@
+import Char from '../utils/Char';
+import Client from '../utils/Client';
+import Clipboard from '../utils/Clipboard';
+import { FetchErrorBox, HorizontalLine } from '../Components';
+import SslCertificate from '../components/SslCertificate';
+
+const SslCertificateError = (props) => {
```

Comment:
Extremely similar to `PortalAccessKeyError`, probably a reasonable common component?